cypress/integration/mailinglist-signup-flow.spec.js

```diff
@@ -224,6 +224,41 @@ describe(`Feature - Mailing list sign up : Tests execution date and time : ${new
 			.should("exist")
 			.should("have.text", "You’ve already signed up");
 	});
+
+	it("It shows Privacy policy details to the user if he clicks on link", function () {
+		mailingListSignUp.getFirstName().type(this.testInputData.firstName);
```

Comment:
May be worth abstracting filling in the first/last/email into a helper since it's rinse and repeated throughout this file?

Reply:
Implemented